### ISSUE TITLE
In Work view, collection branding image in ‘Member of Collection’ section box lacks alt text

### ISSUE THREAD

**Issue by noraegloff** [LafayetteCollegeLibraries/spot#854]
In item view, collection branding image in ‘Member of Collection’ section box at bottom of page lacks alt text. 
`<img>` elements either must have alternate text or a role of none or presentation.

Page tested: https://ldr.lafayette.edu/concern/publications/jh343s88v

Associated elements/source code viewed in test: 
`<img src="/branding/0c483j36g/logo/FacPubs_logo.jpg">`

**Comment by noraegloff** [comment]
Deque aXe tools severity level = **critical**
